src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_STORAGE = 10;
+    private K[] keys;
+    private V[] values;
+    private int arraySize = 0;
```

Comment:
```suggestion
    private int arraySize;
```
size is 0 by default